src/homework/seventh/singleton/Singleton.java

```diff
@@ -0,0 +1,27 @@
+package homework.seventh.singleton;
+
+public final class Singleton {
+    private static Singleton example;
+    private String value;
+
+    public String getValue() {
+        return value;
+    }
+
+    private Singleton() {
```

Comment:
firstly constructors, then other methods
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)